kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OrderRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.time.LocalDate;
+
+public interface OrderRepository
+{
+    public void createOrder(Customer customer, Product product, LocalDate date);
```

Comment:
metody w interfejsach są domyślnie public, można tutaj to usunąć.

Reply:
Zgodnie z rozmową użycie zamierzone.